### DISCUSSION TITLE
Help with FlashForge Adventurer 5m Pro

### DISCUSSION THREAD

**Discussion by JoseLRiveraCruz** [OrcaSlicer/OrcaSlicer#5903] [General]
Anyone operates the Flashforge adventurer 5m pro and has any tips and advices they can give me? I own one and I've had trouble using 3rd party filaments. All I want is some advice on using " Protopasta Stainless Steel Filled PLA 3D Printer Filament | ProtoPlant Metal-Filled PLA Plastic Printer Filament with 60% Metal | Polish and Patina Friendly Metal Composite Filament | 500g Spool. " I want to buy this filament but the advice is to use a rubi tipped nozzle. I don't have one and I don't know there is one for flash forge 5m pro. any advice?

**Comment by Avpman2** [comment]
I think you need to use the .6 or .8 nozzle, assuming either nozzle supports your filament's heat point. You will tear up the .4 nozzle.

**Reply by JoseLRiveraCruz** [reply]
Thanks, Ill keep it in mind

**Reply by Avpman2** [reply]
Let us know how you make out. Good luck!
